src/renderer/api-decorator.js

```diff
@@ -196,17 +197,21 @@
             e.preventDefault();
         });
         global.addEventListener('click', e => {
-            const tag = e.target.tagName;
             const modifiedClick = e.shiftKey || e.metaKey || e.ctrlKey || e.altKey;
-            const mightOpenNewWindow = tag === 'A' || tag === 'IMG';
-            if (mightOpenNewWindow && modifiedClick) {
-                e.preventDefault();
-            } else if (modifiedClick && (tag === 'BUTTON' || tag === 'INPUT')) {
-                if (e.target.type === 'submit') {
+            e.path.some(target => {
+                const tag = target.tagName;
+                const mightOpenNewWindow = tag === 'A' || tag === 'IMG';
+                if (mightOpenNewWindow && modifiedClick) {
                     e.preventDefault();
-                }
+                    return true;
+                } else if (modifiedClick && (tag === 'BUTTON' || tag === 'INPUT')) {
+                    if (target.type === 'submit') {
+                        e.preventDefault();
+                        return true;
```

Comment:
Could this prevent a form from being submitted?

Reply:
Possibly, we already do it though...

Reply:
indeed. ⚓️ 🐛. Are the added `return`s intentional?

Reply:
way to abort the iteration over path